README.md

```diff
@@ -1,20 +1,26 @@
-# swagger1st (swagger first)
+##swagger1st: A Swagger-First Clojure Ring handler
 
 ![Maven Central](https://img.shields.io/maven-central/v/org.zalando/swagger1st.svg)
 [![Build Status](https://travis-ci.org/zalando/swagger1st.svg?branch=master)](https://travis-ci.org/zalando/swagger1st)
 [![codecov](https://codecov.io/gh/zalando/swagger1st/branch/master/graph/badge.svg)](https://codecov.io/gh/zalando/swagger1st)
 
 swagger1st is a Clojure [Ring](https://github.com/ring-clojure/ring) handler that parses, validates and routes requests
-based on your [Swagger](http://swagger.io/) definition. Instead of defining routes and validation rules in your code,
-you specify your API in the [Swagger 2.0 Specification format](https://github.com/swagger-api/swagger-spec) using
-[their great tool set](http://editor.swagger.io/). This encourages an "API first" approach by letting you specify your
-API in a technology independent format. The resulting definition is the ultimate format for publishing, sharing and
-reviewing your API. swagger1st will use it as a configuration file for processing incoming requests. This approach makes
-sure, that your implementation and specification never gets out of sync. During runtime, you can inspect and easily test
-your API with the built-in [Swagger UI](http://petstore.swagger.io/). You are also free to extend the interpretation of
+based on your [Swagger](http://swagger.io/)/OpenAPI definition. Instead of defining routes and validation rules in your code, you can use swagger1st along with [Swagger/OpenAPI's great tool set](http://editor.swagger.io/) to specify your API according to the [Swagger/Open API 2.0 Specification](https://github.com/swagger-api/swagger-spec). This enables you to specify your API in an API-First, technology-independent format. The resulting definition is the ultimate format for publishing, sharing and reviewing your API.
+
+####Compatibility Overview
+swagger1st aims to implement all of the Swagger/OpenAPI spec's features, so that you only have to write your business logic. [This document](https://github.com/zalando/swagger1st/blob/master/comp-2.0.md) shows which aspects of the spec it currently supports.
+
+**[Questions:
```

Comment:
I think highlighting "Questions" is enough. `**` is for inline text only, and thus breaks the format when rendering.